clio-integration-test/src/main/python/clio_test.py

```diff
@@ -29,7 +29,7 @@ def test_wait_for_clio():
             js = r.json()
             clio_status = js['clio']
             search_status = js['search']
-            if clio_status == 'Started' and search_status == 'OK':
+            if clio_status == 'Started' : # and search_status == 'OK':
```

Comment:
https://github.com/broadinstitute/clio/commit/e19ccf61d6b3440ef66a914754599b6fc11b08c5 is a fix for this problem.